src/helpers/validators.js

```diff
@@ -0,0 +1,78 @@
+export const validateHelper = {
+  isEmail: value => {
+    const res = value
+      .toString()
+      .match(
+        /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+      );
+    return res !== null;
+  },
+  isUrl: value => {
+    const res = value
+      .toString()
+      .match(/(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/g);
+    return res !== null;
+  },
+  isNumber: value => {
+    const res = value.toString().match(/\D/);
+    return res === null;
+  },
+  isEmpty: value => value.toString().trim() === "",
```

Comment:
If value is undefined

Reply:
updated with `
isEmpty: value => typeof value == "undefined" || value?.toString().trim() === "",`